accession-commons-core/src/main/java/uk/ac/ebi/ampt2d/commons/accession/hashing/SHA1HashingFunction.java

```diff
@@ -25,6 +25,10 @@
 import java.security.NoSuchAlgorithmException;
 import java.util.function.Function;
 
+/**
+ * Using SHA1 algorithm hashing technique to generate unique accession for the object
```

Comment:
Implementation of Sha1Hashing Function to generate unique hash for a given string object

Reply:
What about this?

```suggestion
 * Implementation of the SHA1 hashing function, in order to generate a unique hash from a given string.
```